modules/acquia_cms_common/acquia_cms_common.info.yml

```diff
@@ -2,7 +2,7 @@ name: "Acquia CMS Common"
 package: "Acquia CMS"
 description: "Handles shared functionality for Acquia CMS."
 type: module
-core_version_requirement: ^9
+core_version_requirement: ^9 || ^10
 dependencies:
   - drupal:acquia_purge
```

Comment:
As part of this ticket, please update .info.yml files w.r.t drupal module dependency listing format `{project}:{module}`.
```
dependencies:
  - acquia_purge:acquia_purge
  - acquia_telemetry:acquia_telemetry
```

Reply:
Thanks @rajeshreeputra!

Reply:
done